include/antler/project/dependency.hpp

```diff
@@ -39,21 +46,41 @@ class dependency {
    /// @param tag  This is either the github repo tag or commit hash, commit hash being preferable. This may be empty if rel is populated or loc points to an archive.
    /// @param rel  This is a github repo version. Using a commit hash tag is preferable. This may be empty.
    /// @param hash  If loc points to an archive, this should be populated with the sha256 hash.
-   void set(std::string_view name, std::string_view loc, std::string_view tag, std::string_view rel, std::string_view hash);
+   void set(std::string name, std::string_view loc, std::string_view tag, std::string_view rel, std::string_view hash);
 
    /// Get the dependency name.
    /// @return The name of this dependency.
-   [[nodiscard]] std::string_view name() const noexcept;
+   [[nodiscard]] inline const std::string& name() const noexcept { return m_name; }
+
+   ///// Get the project name of the dependency.
+   ///// @return The name of the project.
+   //[[nodiscard]] inline const std::string& proj() const noexcept { return m_proj; }
```

Comment:
please remove unnecessary commented out code unless it was done on purpose

Reply:
removed in 213c572c5b06f7e87c4cacacfd40e21c93188a0a